addons/mrp/models/mrp_workcenter.py

```diff
@@ -75,13 +79,69 @@ class MrpWorkcenter(models.Model):
     tag_ids = fields.Many2many('mrp.workcenter.tag')
     capacity_ids = fields.One2many('mrp.workcenter.capacity', 'workcenter_id', string='Product Capacities',
         help="Specific number of pieces that can be produced in parallel per product.", copy=True)
+    kanban_dashboard_graph = fields.Text(compute='_compute_kanban_dashboard_graph')
 
     @api.constrains('alternative_workcenter_ids')
     def _check_alternative_workcenter(self):
         for workcenter in self:
             if workcenter in workcenter.alternative_workcenter_ids:
                 raise ValidationError(_("Workcenter %s cannot be an alternative of itself.", workcenter.name))
 
+    def _compute_kanban_dashboard_graph(self):
+
+        def _get_date_array():
```

Comment:
what's the purpose of an inner function called once? Either move it to the object itself to make it overidable or delete it and inject the code at the correct place

Reply:
leftover from another idea that I forgot to clean up, fixed